### PR TITLE
Tarwa Conglomerate Diona Limbs

### PR DESCRIPTION
**Pull request by RustingWithYou**
Tarwa Conglomerate pirates now have a chance to spawn with nymph limbs.

Sometimes when an arm or leg nymphizes, the child limb doesn't nymphize correctly and just disappears. I couldn't find a way to prevent this from happening, but the player can detach and reattach the limb to fix it, which has been mentioned in the welcome message.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by DreamySkrell** [comment]
I do not think it should be done like this / at all. It means consistent characters sometimes spawn with the diona limbs, sometimes not, with no control if they should/shouldn't.

The best way would be to allow picking diona (or robotic) limbs to the character creator thingy you see when spawning as off-ship char. But that would be a big addition that would not affect just this Tarwa ship.

File: maps/away/away_site/unathi_pirate/tarwa/unathi_pirate_tarwa_ghostroles.dm [hunk 1]
```diff
@@ -56,13 +56,29 @@
 		/obj/item/storage/box/survival = 1
 	)
 
+/datum/outfit/admin/tarwa/post_equip(mob/living/carbon/human/H, visualsOnly)
+	if(!istype(H))
+		return
+	for(var/name in H.organs_by_name)
+		var/obj/item/organ/external/O = H.organs_by_name[name]
+		if(!O || name == BP_HEAD || name == BP_CHEST || name == BP_GROIN)
+			continue
```
Comment (NonQueueingMatt) [inline maps/away/away_site/unathi_pirate/tarwa/unathi_pirate_tarwa_ghostroles.dm:65]: You have an issue here with the loop variable `name` overlapping with the ghostspawner `name` variable.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by RustingWithYou** [comment]
> I do not think it should be done like this / at all. It means consistent characters sometimes spawn with the diona limbs, sometimes not, with no control if they should/shouldn't.
> 
> The best way would be to allow picking diona (or robotic) limbs to the character creator thingy you see when spawning as off-ship char. But that would be a big addition that would not affect just this Tarwa ship.

I agree, but that is a lot more work and I don't know how to do it. Overall it's a fairly minor blow to consistency while adding some more of the faction's specific thing to its ghostrole IMO.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**